### PR TITLE
Remove One Intake Solenoid

### LINKED ISSUE #17: Remove second solenoid from Intake subsystem
Physically, the robot uses a single DoubleSolenoid to control both intakes (Front and back). So we can remove the second `DoubleSolenoid` from our subsystem and from the RobotMap

### PR DESCRIPTION
**Pull request by Jacob4649**
Fix #17

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -78,7 +78,6 @@ private void configureButtonBindings() {
     JoystickButton climbButton = new JoystickButton(m_driveJoystick, RobotMap.JOYSTICK_BUTTON.CLIMB);
 
     JoystickButton frontIntake = new JoystickButton(m_driveJoystick, RobotMap.JOYSTICK_BUTTON.FRONT_INTAKE);
```
Comment (matthew-reynolds) [inline src/main/java/frc/robot/RobotContainer.java]: Please rename this button and RobotMap constant. "Front" is no longer needed.

**Review by matthew-reynolds — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (matthew-reynolds) [reply src/main/java/frc/robot/RobotContainer.java]: Resolved this in 9ca9bbb

Comment (Jacob4649) [reply src/main/java/frc/robot/RobotContainer.java]: Resolved in 9ca9bbb and 4efc653

**Review by matthew-reynolds — APPROVED** [review]
LGTM

**Pull request merged**